packages/communication-react/review/communication-react.api.md

```diff
@@ -1647,27 +1647,53 @@ export interface MessageThreadStyles extends BaseCustomStyles {
 // @public
 export const MicrophoneButton: (props: MicrophoneButtonProps) => JSX.Element;
 
+// @public
+export interface MicrophoneButtonContextualMenuStyles extends IContextualMenuStyles {
+    menuItemStyles?: IContextualMenuItemStyles;
+}
+
 // @public
 export interface MicrophoneButtonProps extends ControlBarButtonProps {
+    microphones?: OptionsDevice[];
```

Comment:
Is there a way that we can tell whether the api docs is beta flavor or not? I am guessing that this is beta since it has your new props but is github that smart?

Reply:
We generate two different api docs now -- one for each flavor. This one only updates the default (beta) one.